src/config/backpack/basset.php

```diff
@@ -4,6 +4,9 @@
     // development mode, assets will not be internalized
     'dev_mode' => env('BASSET_DEV_MODE', env('APP_ENV') === 'local'),
 
+    // verify ssl certificate while fetching assets
+    'verify_ssl_certificate' => true,
```

Comment:
Wasn't the initial intention of this to provide a way of skip the cert. validation in some envs ?
In that case I think we need a env variable right ? 
```suggestion
    'verify_ssl_certificate' => env('BASSET_VERIFY_SSL_CERTIFICATE', true),
```